### PR TITLE
test on GROMACS 2022 and 2023

### PR DESCRIPTION
**Pull request by orbeckst**
- add conda-forge GROMACS 2022 and 2023 to CI
- update README and INSTALL
- update docs (supported versions of GROMACS)
- update CHANGES

### CODE REVIEW

**Comment by orbeckst** [comment]
Once the CI is all green, please have a quick look.

If changes are needed, do a *request changes* review and state what needs to be done.

Otherwise do a *approve* "LGTM". 

Thanks.

**Comment by orbeckst** [comment]
Everything passed on the first attempt so I assume we can merge this soon.

I changed the branch protection settings so that the new 2023.1 tests are now required.

File: INSTALL.rst
Comment (cadeduckworth) [inline INSTALL.rst:35]: seaborn is missing

Comment (cadeduckworth) [inline INSTALL.rst:35]: add seaborn, which is already included in `MDPOW/doc/requirements.txt`

**Review by cadeduckworth — CHANGES_REQUESTED** [review]

**Review by VOD555 — APPROVED** [review]
LGTM

File: INSTALL.rst
Comment (orbeckst) [reply INSTALL.rst]: thanks, addressed in 301ef52c2b55be7ed78acd61b196a8dcf5af3eea

**Review by iorga — APPROVED** [review]
LGTM

**Comment by orbeckst** [comment]
@cadeduckworth please have a look, I addressed your changes.

Once you're happy, approve, and then you can **merge** the PR. (After your approval, everything should be green and the green merge button should appear for you. You can just *merge* the PR as all commits are self contained. On more complicated PRs we would *squash merge*.)

**Review by cadeduckworth — APPROVED** [review]

**Comment by cadeduckworth** [comment]
@orbeckst done!

**Pull request merged**
